lib/ae_mdw/stats.ex

```diff
@@ -243,23 +251,44 @@ defmodule AeMdw.Stats do
 
   defp build_transactions_statistics_streamer(state, query, _scope, cursor) do
     tx_tag = Keyword.get(query, :tx_type, :all)
+    interval_by = Keyword.get(query, :interval_by, :day)
 
     cursor =
       case cursor do
         nil -> nil
-        interval_start -> {{:transactions, tx_tag}, :day, interval_start}
+        interval_start -> {{:transactions, tx_tag}, interval_by, interval_start}
       end
 
     key_boundary = {
-      {{:transactions, tx_tag}, :day, Util.min_int()},
-      {{:transactions, tx_tag}, :day, Util.max_int()}
+      {{:transactions, tx_tag}, interval_by, Util.min_int()},
+      {{:transactions, tx_tag}, interval_by, Util.max_int()}
     }
 
     fn direction ->
       Collection.stream(state, Model.Statistic, direction, key_boundary, cursor)
     end
   end
 
+  defp render_statistic(state, {{:transactions, _tx_tag}, :month, interval_start} = statistic_key) do
+    Model.statistic(count: count) = State.fetch!(state, Model.Statistic, statistic_key)
+
+    %{
+      start_date: months_to_unix(interval_start),
```

Comment:
how about using an iso date instead of the timestamp? It's compact, makes it already formatted to render on the frontend and is also human readable.

Reply:
this is how the node formats time already, so I'd thought I wouldn't change it to avoid having the client implement two separate decoders for datetimes

Reply:
With ISO date there is no need for decoders, it's yyyy-mm-dd. Even though the issue says `start_time`, the name chosen in the PR `start_date` makes it more precise since there is no time related to this stats.